sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -120,3 +121,34 @@ export enum DesignationStatus {
    */
   NotDesignated = "Not designated",
 }
+
+export class ScholasticStandingAPIInDTO {
+  @Allow()
+  scholasticStanding: string;
+  @Allow()
+  dateOfChange?: string;
```

Comment:
From the API perspective there is no difference between `dateOfChange`, `dateOfCompletion`, `dateOfIncompletion`, `dateOfWithdrawal` can we keep the same API value on form.io and make the API receive just one `offeringEndDate`, for instance?

Reply:
yes, the API perspective ists the same, but we can't use same api values multiple times in form definitions. Also, if we are using same fields then according to Figma the label/title of each date is different 
![image](https://user-images.githubusercontent.com/77353155/163853790-1b4ffdeb-d6aa-44b4-8b35-becf75e55939.png)
![image](https://user-images.githubusercontent.com/77353155/163853813-27ddee44-d0f2-45dd-b33a-ad1b56cb54e9.png)
![image](https://user-images.githubusercontent.com/77353155/163853848-abafd451-a1da-46ec-93ff-5df09ca60bc9.png)
![image](https://user-images.githubusercontent.com/77353155/163853888-b7b38772-90f4-4874-9fd1-b3b00c7c6b1c.png)